### ISSUE TITLE
Is it possible to use the crop feature without the "fill" option?

### ISSUE THREAD

**Issue by nifel87** [wagtail/wagtail#4075]
Hi everyone, recently I'm trying to use the crop feature of wagtail, but if I understand correctly the documentation, it can be use only in combo with the fill option?

I got it wrong or it is just so?

Thanks for your time, best regards,
nifel87

**Comment by gasman** [comment]
Hi @nifel87,
If you have a question about whether something is possible in Wagtail,
 please ask it on [Stack Overflow](https://stackoverflow.com/questions/tagged/wagtail). However, in this case, no, it isn't possible, so I'll consider this a feature request...

What are you trying to do that can't be achieved by an `{% image foo fill-400x300 %}` tag with your desired dimensions?

**Comment by nifel87** [comment]
Sorry for the late reply. Basicly I've tried to crop an entire bunch of images using only their focal area without resizing them. This because all the images had a thick yellow border. So I decided to crop them systematically in order to remove the border.

Thing is that I can't crop them without resizing them and the system as it is now doesn't allow to crop an image to exactly the dimensions of the focal area.
